src/github/mod.rs

```diff
@@ -189,6 +196,48 @@ impl SyncGitHub {
         Ok(diffs)
     }
 
+    fn diff_toml_teams(&self) -> anyhow::Result<Vec<OrgMembershipDiff>> {
+        let mut diffs = Vec::new();
+        for team in &self.teams {
+            diffs.push(self.diff_team_gh_org(team)?);
+        }
+        Ok(diffs)
+    }
+
+    // create diff against github org members and toml team members
+    fn diff_team_gh_org(
+        &self,
+        team: &rust_team_data::v1::Team,
+    ) -> anyhow::Result<OrgMembershipDiff> {
+        let toml_team_members = &team.members;
+        let team_org;
+        // get the team github org through the corresponding github team
+        if let Some(gh) = &team.github {
+            let github_teams = &gh.teams;
+            let github_team: &GitHubTeam = github_teams
+                .iter()
+                .find(|&gt| gt.name == team.name)
+                .expect("Team Not Found");
+            team_org = github_team.org.clone();
+        } else {
+            team_org = "rust_lang".to_string();
```

Comment:
why this value?
Should we report an error if the team doesn't have any associated org?
Also the rust org is called "rust-lang"